StreamChatUITestsAppUITests/Tests/Performance/ScrollTime.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+
+@available(iOS 15.0, *)
+class ScrollTime: StreamTestCase {
+    
+    func testMessageList() {
+        WHEN("user opens the message list") {
+            backendRobot.generateChannels(count: 1, messagesCount: 200)
```

Comment:
Not sure if it's possible with the tooling, but here's an idea to test: Generate first 25 messages, scroll up few times, and then the backend robot generates 25 more. We will measure appending those messages while scrolling, which can also cause a frame drop.

Reply:
You described the pagination logic, didn't you? If so, this is how it works at the moment 🤔